Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+pipeline {
+  agent {
+    label 'sbt-slave'
+  }
+
+  environment {
+    // Ensure that build scripts recognise the environment they are running within
+    CI = ''
+    SCALA_VERSION = '2.10.6'
```

Comment:
I disagree. The internal version of scala for the build tool is an implementation detail and all we're doing here is introducing complexity and duplication for no good reason.